pallene/ir.lua

```diff
@@ -33,6 +33,7 @@ function ir.Module()
         globals            = {}, -- list of ir.VarDecl
         exported_functions = {}, -- list of function ids
         exported_globals   = {}, -- list of variable ids
+        lambdas            = {}, -- { function id }
```

Comment:
If we stop adding lambdas to the `exported_functions` list, can we get rid of this `module.lambdas` map?

Reply:
I'm not sure if we can do that.
In `coder.lua`, we are recursively going through every single function and adding it to the "closures" list. (line 675)